components/usage-api/usage/v1/billing.proto

```diff
@@ -19,6 +19,9 @@ service BillingService {
   // FinalizeInvoice marks all sessions occurring in the given Stripe invoice as
   // having been invoiced.
   rpc FinalizeInvoice(FinalizeInvoiceRequest) returns (FinalizeInvoiceResponse) {};
+
+  // SetBilledSession marks an instance as billed with a billing system
+  rpc SetBilledSession(SetBilledSessionRequest) returns (SetBilledSessionResponse) {};
```

Comment:
From reading the RPC, it's also unclear how `SetBillSession` will behave if such a record already exists, how if it doesn't. I think it will do an upsert. But it's fine to leave as is, it's an internal API.